schemas/latest/landingzones/types/appservice.json

```diff
@@ -0,0 +1,34 @@
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
```

Comment:
Since it's a specific implementation of app service (i.e. Linux + Containers), can this type be renamed as "appservice-linux-container" to be more precise?

Reply:
Renamed as appServiceLinuxContainer per camelCase naming convention.